frontend/src/sass/elements/_buttons.scss

```diff
@@ -160,3 +160,8 @@
   border-radius: 4px;
   padding: 0 4px;
 }
+
+.usa-button-disabled {
+  opacity: 65%;
+  margin-top: calc(3rem - .2em);
+}
```

Comment:
We can get rid of this if we use the `disabled` attribute  mentioned in the other comment.

Reply:
Beautiful. I'll axe it.